### ISSUE TITLE
Steam Overlay does not open when it and Awakened Trade are both bound to F7

### ISSUE THREAD

**Issue by Razgriz450** [KloppstockBw/GFNPoEPriceCheck#13]
Posted on your YT originally but figured it might be better to make a post here. Steam Overlay is not opening when I press F7 while Awakened PoE trade is also bound to F7. Steam Overlay will only open when I hold F7. If I change the Awakened Trade hotkey and press F6 the script does copy the item info to the google doc and tries to paste to the Price Check search bar which does not open. I can't find any settings in Steam or Awakened Trade that I think would cause this interaction.

I've recorded my display to show you what it looks like on my screen
https://streamable.com/u864x5

**Comment by KloppstockBw** [comment]
Awakened trade needs to stay at default ctrl D

**Comment by Razgriz450** [comment]
> Awakened trade needs to stay at default ctrl D

Ok I have made this change and Steam overlay does open with F7 but Awakened Trade still does not open when the script tries to past the item info to the Price Check search bar. I've reset the config several times and ensured the position was set correctly. I noticed Awakened trade doesn't appear when I do set the position but not sure if that's how its supposed to be. Feels like I missed something simple or maybe there's a setting I don't know about?
